libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationViewModel.java

```diff
@@ -26,6 +26,7 @@
 import com.mapbox.navigation.base.typedef.TimeFormatType;
 import com.mapbox.navigation.core.MapboxDistanceFormatter;
 import com.mapbox.navigation.core.MapboxNavigation;
+import com.mapbox.navigation.core.directions.session.RoutesObserver;
```

Comment:
All changes in this PR except for `RoutesObserver` is already addressed in this [PR](https://github.com/mapbox/mapbox-navigation-android/pull/2527)